src/models/hooks/activityReportObjective.test.js

```diff
@@ -131,4 +131,44 @@ describe('activityReportObjective hooks', () => {
       expect(transaction.finished).toBe('commit');
     });
   });
+
+  describe('propagateSupportTypeToObjective', () => {
+    let supportObjective;
+    let aroWithSupportType;
+
+    beforeAll(async () => {
+      supportObjective = await Objective.create({
+        title: 'test support objective',
+        status: OBJECTIVE_STATUS.NOT_STARTED,
+      });
+    });
+
+    afterAll(async () => {
+      await Objective.destroy({
+        where: { id: supportObjective.id },
+        force: true,
+      });
+
+      if (aroWithSupportType) {
+        await ActivityReportObjective.destroy({
+          where: { id: aroWithSupportType.id },
+        });
```

Comment:
I know I approved this but does this work? deleting the objective while there is still an ARO attached to it? I think you'd have to do the aro destroy first... you can also lose the if and do ```where: { objectiveId: supportObjective.id }```, I assume

Reply:
Oh yeah, good catch: 657a9c117349f7f69b7122eb12b2c572c5e0a6c6